File: Day 1/Java/TaskFive.java
import java.util.Scanner;
public class TaskFive{
public static void main(String[] args){

Scanner input = new Scanner(System.in);

int ScoreCounter = 0;
int sumOfScores = 0;
int numberOfStudents = 10;
int evenNumber = 2;
int sumOfEvenNumbers = 0;
while (ScoreCounter < 10){

System.out.print("Enter a score: ");
int score = input.nextInt();

sumOfScores += score;

ScoreCounter++;

if(score % evenNumber == 0) sumOfEvenNumbers += score;

}

double average = sumOfScores / numberOfStudents;


System.out.print("The sum of the even indexes is "+ sumOfEvenNumbers);



}
}

File: Day 1/Python/TaskFive.py


Score_Counter = 0
sum_Of_Scores = 0
number_Of_Students = 10
even_Number = 2
sum_Of_Even_Numers = 0

while Score_Counter < 10:

	score = int(input("Enter a score: "))

	sum_Of_Scores  += score;

	Score_Counter = Score_Counter + 1

	if score % even_Number == 0:
		sum_Of_Even_Numers += score

average = sum_Of_Scores / number_Of_Students


print(f'The sum of even numbers is {sum_Of_Even_Numers}');

